src/Faithlife.Build/DotNetClassicTool.cs

```diff
@@ -72,7 +72,7 @@ public sealed class DotNetClassicTool
 
 		var bestMatch = VersionRange.Parse(packageVersion).FindBestMatch(packageDirectories) ??
 			throw new BuildException($"Found restored NuGet package folder but no version is a best match: {packageFullPath}");
-		var packageBestMatchPath = Path.Combine(packagesPath, packageName.ToLowerInvariant(), bestMatch.OriginalVersion);
+		var packageBestMatchPath = Path.Combine(packagesPath, packageName.ToLowerInvariant(), bestMatch.OriginalVersion ?? bestMatch.ToString());
```

Comment:
This was changed due to a nullability analysis warning. Not sure if `!` would have worked OK here (i.e., if `OriginalVersion` is guaranteed to be non-null with the way we initialize it).